src/couch/src/couch_secondary_sup.erl

```diff
@@ -26,6 +26,16 @@ init([]) ->
             worker,
             dynamic}
     ],
+    Daemons = [
+        {"index_server", "{couch_index_server, start_link, []}"},
```

Comment:
It seems a bit silly to keep these as strings just to convert all of the terms in the following list comprehension. I'd at least like to remove that bit of silly for this supervisor. I'd also suggest that we just write out the child specs directly in the SecondarySupervisors list above given that's the more traditional way of doing things (or at least using a helper that gets mapped over the name/mfa pairs.

Reply:
Latest push removes the redundant string parsing. I’d accept commits that turn the list comprehension into a helper, or one that writes it all out.